changelog/issue-3163.md

```diff
@@ -0,0 +1,6 @@
+audience: general
+level: silent
```

Comment:
Don't think silent level writes out to the changelog (may be wrong), probably better as patch I think.